src/include/msgr.h

```diff
@@ -84,14 +84,15 @@ struct ceph_entity_name {
 	__le64 num;
 } __attribute__ ((packed));
 
-#define CEPH_ENTITY_TYPE_MON    0x01
-#define CEPH_ENTITY_TYPE_MDS    0x02
-#define CEPH_ENTITY_TYPE_OSD    0x04
-#define CEPH_ENTITY_TYPE_CLIENT 0x08
-#define CEPH_ENTITY_TYPE_MGR    0x10
-#define CEPH_ENTITY_TYPE_AUTH   0x20
```

Comment:
@liewegas @tchaikov any particular reason why `type` is a bit-field? Should it not just be consecutive integers?

i.e. I think it's nonsensical for a type to be CEPH_ENTITY_TYPE_MON|CEPH_ENTITY_TYPE_OSD.